common/helpers.js

```diff
@@ -76,6 +76,7 @@ function dataObjectUriToHttps(dataObjectUri) {
 const BondProviders = Object.freeze({
     FENCE: 'fence',
     DCF_FENCE: 'dcf-fence',
+    HCA: 'hca', // Human Cell Atlas
```

Comment:
This implies that there is a bond provider for HCA, which according to the description there isn't. It might be clearer to remove this, return `null` from `determineBondProvider`, and make the bond call conditional on the existence of the provider string.

Reply:
At first glance, I agreed with you @panentheos , but now that I had a chance to talk this over with @abaumann and @geoffjentry , I'm OK with it.  Eventually, HCA will have a real provider.  Due to limitations in the DOS spec at the moment, it's hard to know which provider to use for which DOS URIs.  This at least keeps all that logic isolated to Martha.

Reply:
@panentheos, @gplumbo-broad Thank you both for your review and comments.
Indeed, this is just a first, tiny step towards a bona fide Bond provider for HCA that will be developed over time. The significance of the Bond provider to the Terra/HCA interaction is clearly evident in this design document [Terra/Cromwell Access to Files Identified by DRS URIs for the HCA/Terra MVP](https://docs.google.com/document/d/18WDYr3uS82Eu1GhHmeH6PX7tFY6Zk8piKd3ixOoE8Bg)

The immediate objective is to enable Terra to access HCA for the HCA handoff to Terra MVP initiative. Currently, all HCA data is open/public access, and anyone can obtain a presigned URL for any data. However, Cromwell currently requires cloud-native URLs, so HCA is providing special access to the Broad (Mint and Terra) via cloud-native URLs.  HCA will be supporting controlled access data with user-specific authN/authZ over time and is developing the Fusillade system to support this.
Although currently, HCA data is open/public access, the bucket containing it is access controlled. For the immediate/near-term, the recent plan had been to develop a minimal HCA provider based on the `mock-provider` to provide a single/fixed HCA service account key for all Terra users. Then @abaumann proposed the simpler, easier short-term solution of simply adding  `GROUP_All_Users@firecloud.org` with read access to the ACL of the HCA bucket, ... and hence this PR. :)